### PR TITLE
#216 Range

### PR DESCRIPTION
**Pull request by VsSekorin**
As per #216 
`IntRange` inroduced

### CODE REVIEW

**Comment by 0crat** [comment]
@vssekorin I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/main/java/org/cactoos/list/IntRange.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+/**
+ * Range of integer number (Z).
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @author Vseslav Sekorin (vssekorin@gmail.com)
+ * @version $Id$
+ * @since 0.8
+ */
+public final class IntRange implements Iterable<Long> {
```
Comment (fabriciofx) [inline src/main/java/org/cactoos/list/IntRange.java]: @VsSekorin Is it not strange a class called `IntRange` and it returns a `Long`?

Comment (VsSekorin) [inline src/main/java/org/cactoos/list/IntRange.java]: [Integer number (Z)](https://en.wikipedia.org/wiki/Integer). Long contains integer numbers.

Comment (fabriciofx) [inline src/main/java/org/cactoos/list/IntRange.java]: @VsSekorin Yeah, I know, but if I'm using `IntRange` I'd like get a range of `Integer`s, right? In other words, the problem is the name. Maybe `NumberRange` makes more senses. WDYT?

**Comment by yegor256** [comment]
@VsSekorin it seams that we can use `new LimitedIterable(new SkippedIterable(new NaturalNumbers()))` to achieve the same goal, right?

**Comment by VsSekorin** [comment]
@yegor256 Yeah

**Pull request closed without merging**